playbooks/ReadMe.txt

```diff
@@ -0,0 +1,9 @@
+Automate provisioning OpenStack Instance
+
+- The Ansible playbook will manually create the OpenStack instance without having to open a virtual machine. 
```

Comment:
"will automate the creation of an openstack instance." 
@LikhithaEda

Reply:
You need not have separate Readme.txt 
you can always include the contents inside playbook as comments at the top
@LikhithaEda